src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetCoreMvc2Integration.cs

```diff
@@ -263,7 +268,9 @@ public AspNetCoreMvc2Integration(object actionDescriptor, object httpContext)
         [InterceptMethod(
             CallerAssembly = "Microsoft.AspNetCore.Mvc.Core",
             TargetAssembly = "Microsoft.AspNetCore.Mvc.Core",
-            TargetType = ResourceInvoker)]
+            TargetType = ResourceInvoker,
```

Comment:
I know this isn't new, but is there a reason this isn't referenced by full name, like `Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker`?

Reply:
Ah yeah, it is the fully qualified name, it's in the const higher up in the file.